src/main/java/winter/Manager.java

```diff
@@ -0,0 +1,302 @@
+package winter;
+
+import java.util.Scanner;
+import winter.checkedexceptions.EmptyCommandException;
+import winter.checkedexceptions.InvalidCommandException;
+import winter.checkedexceptions.InvalidDeadlineException;
+import winter.checkedexceptions.InvalidToDoException;
+import winter.checkedexceptions.InvalidEventException;
+import winter.task.Deadline;
+import winter.task.Event;
+import winter.task.Task;
+import winter.task.ToDo;
+
+import static java.sql.Types.NULL;
+import static winter.Commands.*;
+
+
+public class Manager {
+    private static final String line = "-----------------------------------\n";
+    private static final String indent = "   ";
+    private static int taskIndex = 0;
+    private static final Task[] taskList = new Task[100];
+
+    public static void acceptInput () {
+        Scanner input = new Scanner(System.in);
+        String inputString;
+        Commands action;
+        boolean flag = true;
+
+        do {
+            inputString = input.nextLine();
+            action = classifyCommand(inputString);
+
+            int spaceIndex = inputString.indexOf(" ");
+            int slashIndex = inputString.indexOf("/");
+
+            if (slashIndex == -1) {
+                slashIndex = inputString.length();
+            }
+
+            switch (action) {
+            case TODO:
+            case DEADLINE:
+            case EVENT:
+                addTask(action,inputString.substring(spaceIndex,slashIndex),inputString,slashIndex);
+                continue;
+            case BYE:
+                sayBye();
+                flag = false;
+                break;
+            case LIST:
+                displayList(taskList, taskIndex);
+                continue;
+            case MARK:
+                taskList[Integer.parseInt(inputString.substring(5)) - 1].mark();
```

Comment:
Avoid magic numbers like this as it make not be immediately apparent what `5` means